libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -106,6 +106,13 @@ func (m *Manager) Apply(pid int) (err error) {
 		return nil
 	}
 
+	var c = m.Cgroups
+
+	if c.Paths != nil {
+		m.Paths = c.Paths
+		return cgroups.EnterPid(m.Paths, pid)
+	}
+
```

Comment:
EnterPid enters the cgroups only if the path exists and that is current behavior for exec. I think it makes sense to have the same behavior here as well.